priv/sample/rebar.config

```diff
@@ -1,17 +1,11 @@
 {erl_opts, [debug_info]}.
-{deps, [ {meck, "0.9.0"} ]}.
+{ct_readable, true}.
 
 {shell, [
   % {config, "config/sys.config"},
     {apps, [sample]}
 ]}.
 
-%% TODO: Windows variant
-{pre_hooks, [ {compile, "rm -rf _build/default/plugins/rebar3_bsp"}
-            , {compile, "ln -sf $(readlink -f ../../) _build/default/plugins/rebar3_bsp"}
-            ]
-}.
-{project_plugins, [ {erl_subgraph_compile, {git, "https://github.com/ferd/erl_subgraph_compile.git", {branch, "main"}}}
-                    %% The rebar3_bsp will be symlinked, so just use a placeholder here
-                  , {rebar3_bsp, {git, "dummy", {branch, "dummy"}}}
-                  ]}.
+{deps, [ {meck, "0.9.0"} ]}.
+{plugins, [rebar3_path_deps, {rebar3_bsp, {path, "../.."}}]}.
```

Comment:
Does this require manual setup by the user? Not familiar with the plugin...

Reply:
No, it works out of the box. The plugin is documented upstream in rebar3: https://rebar3.readme.io/docs/using-available-plugins#vendoring-dependencies

Reply:
We should later on publish the `rebar3_bsp` plugin instead of relying on a path?